qiskit_aer/backends/aerbackend.py

```diff
@@ -386,12 +388,24 @@ def _assemble(self, circuits, parameter_binds=None, **run_options):
                 parameterizations = self._convert_binds(circuits, parameter_binds)
                 qobj = None
                 for circuit in circuits:
-                    assemble_bind = {param: 1 for param in circuit.parameters}
+                    # Replace parameters to one
+                    original_parameters = circuit._parameters
+                    circuit._parameters = []
+                    original_params = []
+                    for instruction in circuit._data:
+                        params = instruction.operation.params
+                        original_params.append(params.copy())
```

Comment:
If there is no `ParameterExpression`, you don't need to make a copy. We may get some performance improvement to avoid copies.

Reply:
Thanks. Good suggestion :+1: